pkg/migrations/sql/idempotent/tracing-functions.sql

```diff
@@ -242,18 +242,18 @@ RETURNS TABLE
     cnt bigint
 )
 AS $func$
-	SELECT
-		parent.operation_id as parent_operation_id,
-		child.operation_id as child_operation_id,
-	  	count(*) as cnt
-	FROM
-		_ps_trace.span child
-	INNER JOIN
-		_ps_trace.span parent ON (parent.span_id = child.parent_span_id AND parent.trace_id = child.trace_id)
-	WHERE
-      		child.start_time > _start_time_min AND child.start_time <_start_time_max AND
-      		parent.start_time > _start_time_min AND child.start_time < _start_time_max
-	GROUP BY parent.operation_id, child.operation_id
+    SELECT
+        parent.operation_id as parent_operation_id,
+        child.operation_id as child_operation_id,
+        count(*) as cnt
+    FROM
+        _ps_trace.span child
+    INNER JOIN
+        _ps_trace.span parent ON (parent.span_id = child.parent_span_id AND parent.trace_id = child.trace_id)
+    WHERE
+        child.start_time > _start_time_min AND child.start_time < _start_time_max AND
+        parent.start_time > _start_time_min AND parent.start_time < _start_time_max
```

Comment:
This isn't a change in the PR, but do we not have an including bound on the lower end of time? I think projects that have time-based responses have this, including our find_traces API.

Basically,
```postgresql
 child.start_time >= _start_time_min AND child.start_time < _start_time_max AND
 parent.start_time >= _start_time_min AND parent.start_time < _start_time_max
```

Reply:
@Harkishen-Singh I thought the same thing. I've always used inclusive-exclusive, however @cevian asked me to check how Jaeger does it, since this function is meant to be Jaeger compatible, and he thought it was exclusive-exclusive. I could definitely be wrong, but the Jaeger code appears to be exclusive-exclusive to me.